package.json

```diff
@@ -5,30 +5,27 @@
   "version": "0.16.11",
   "publisher": "mtxr",
   "license": "GPL-3.0",
-  "main": "dist/extension.js",
+  "main": "../dist/extension.js",
   "repository": {
     "url": "https://github.com/mtxr/vscode-sqltools",
     "type": "git"
   },
   "engines": {
-    "vscode": "^1.28.0"
+    "vscode": "^1.30.0"
   },
   "private": true,
   "workspaces": [
     "packages/*"
   ],
   "scripts": {
-    "clean": "rm -rf dist &> /dev/null",
+    "clean": "rm -rf ../dist &> /dev/null",
     "precompile": "yarn run clean",
-    "prewatch": "yarn run clean",
+    "prewatch": "yarn run clean && mkdir -p ../dist",
     "watch": "yarn workspace @sqltools/extension watch",
     "compile": "yarn workspace @sqltools/extension compile",
     "compile:preview": "yarn workspace @sqltools/extension compile:preview",
-    "package:preview": "yarn run compile:preview && (cd dist && vsce package)",
-    "package": "yarn run compile && (cd dist && vsce package)",
-    "publish": "cd dist && vsce publish"
-  },
-  "dependencies": {
-    "applicationinsights": "^1.1.0"
+    "package:preview": "yarn run compile:preview && (cd ../dist && vsce package)",
+    "package": "yarn run compile && (cd ../dist && vsce package)",
+    "publish": "cd ../dist && vsce publish"
```

Comment:
```suggestion
    "publish": "cd ../dist && vsce publish --yarn"
```

Reply:
What is the reason of putting dist outside of workspace dir?

Reply:
None for now, at least I'm not sure if still needed. 

But one of the attempts made me move outside. Because when I ran npm install during the packing, it was changing the lock file and the node_modules of the mono repository